Packs/DuoAdminApi/README.md

```diff
@@ -0,0 +1,2 @@
+# Duo Admin
+This pack includes Cortex XSIAM content.
```

Comment:
Don't you need some additional information here?

Reply:
In this pack's specific case- No, since it's with an Event Collector integration.